lib/theme-options/site-options.php

```diff
@@ -24,7 +24,7 @@
       'name'    => __( 'Evento Actual/Siguente', 'IGV' ),
       'id'      => $prefix . 'radio_evento_current',
       'type'    => 'select',
-      'show_option_none' => false,
+      'show_option_none' => true,
```

Comment:
This select doesn't populate according to the stated logic on my local. I set the time in the picker to the current time in CDMX and it doesn't show. Nor for any time in the future in CDMX. Not actually code added in this branch but a clear issue.